plugins/woocommerce/changelog/55442-fix-35166-error-downloadable-product-image-viewing

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Avoid display errors for images added as product downloads by showing the placeholder image instead for apache server configuration where access to woocommerce_uploads directly is not allowed.
```

Comment:
Optional thing/quibble: the underlying problem is not exclusive to Apache.

```suggestion
Improve handling of placeholder images for product downloads in cases where server restrictions prevent direct access to the relevant files.
```

Reply:
100% I agree, we serve all files from WooCommerce uploads in this way irrespective of restrictions and limit access to them for everyone besides admin.